bucket/registryworkshop.json

```diff
@@ -0,0 +1,54 @@
+{
+    "description": "An advanced registry editor.",
+    "homepage": "http://www.torchsoft.com/en/rw_information.html",
+    "license": "Shareware",
+    "url": "http://www.torchsoft.com/download/RegistryWorkshop.exe#/dl.7z",
+    "version": "5.0.1",
+    "hash": "cabbb998557c5e883200d082853ca0db1a9f0db66332a0731019b7485c2e95d0",
+    "architecture": {
+        "64bit": {
+            "bin": [
+                [
+                    "RegWorkshopX64.exe",
+                    "regworkshop"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "RegWorkshopX64.exe",
+                    "Registry Workshop"
+                ]
+            ]
+        },
+        "32bit": {
+            "bin": [
+                [
+                    "RegWorkshop.exe",
+                    "regworkshop"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "RegWorkshop.exe",
+                    "Registry Workshop"
+                ]
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\RegWorkshop.ini\")) { New-Item \"$dir\\RegWorkshop.ini\" -ItemType File > $Null}",
+        "if (!(Test-Path \"$persist_dir\\favorites.dat\")) { New-Item \"$dir\\favorites.dat\" -ItemType File > $Null}"
```

Comment:
```suggestion
        "if (!(Test-Path \"$persist_dir\\favorites.dat\")) { New-Item \"$dir\\favorites.dat\" -ItemType File | Out-Null}"
```

Reply:
[null - What's the better (cleaner) way to ignore output in PowerShell? - Stack Overflow](https://stackoverflow.com/questions/5260125/whats-the-better-cleaner-way-to-ignore-output-in-powershell)

Reply:
Why is this bad?

Reply:
No reason to use bash like style of redirection output. Powershell is using pipes, pipes are used in all code base, 

Just nonsense to save few milliseconds.

Reply:
OK, accept.